packages/components/src/components/popover/popover.jsx

```diff
@@ -21,6 +21,7 @@ const Popover = ({
     id,
     is_open,
     is_bubble_hover_enabled,
+    is_quickStrategy,
```

Comment:
Quick Strategy is a part of the DBot project and popover is a common component. so it's better to use a generic name for this prop instead of using a project-related name